From a83d897751e1eb935fe3e32d6179f8f8b8cca16a Mon Sep 17 00:00:00 2001
From: IAP Desktop Build <iap-desktop+build@google.com>
Date: Thu, 16 Dec 2021 08:13:32 +0100
Subject: [PATCH 7/8] Include symbols in NuGet package

---
 VtNetCore/VtNetCore.csproj | 1 +
 1 file changed, 1 insertion(+)

diff --git a/VtNetCore/VtNetCore.csproj b/VtNetCore/VtNetCore.csproj
index b0ca834..6cfd1c8 100644
--- a/VtNetCore/VtNetCore.csproj
+++ b/VtNetCore/VtNetCore.csproj
@@ -13,6 +13,7 @@
     <RepositoryType>Github</RepositoryType>
     <PackageLicenseUrl>https://github.com/darrenstarr/VtNetCore/blob/master/LICENSE</PackageLicenseUrl>
     <PackageTags>VT-100 VT220 xterm NetCore</PackageTags>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="System.Text.Encoding" Version="4.3.0" />
-- 
2.17.1.windows.2

