From 9a9f330f6ba5c67f56b38e0bbf01439862472d26 Mon Sep 17 00:00:00 2001
From: IAP Desktop Build <iap-desktop+build@google.com>
Date: Tue, 11 May 2021 15:52:50 +0200
Subject: [PATCH 2/2] Fix escape sequences for bracketed text mode

---
 VtNetCore/VirtualTerminal/VirtualTerminalController.cs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/VtNetCore/VirtualTerminal/VirtualTerminalController.cs b/VtNetCore/VirtualTerminal/VirtualTerminalController.cs
index cd24780..248c2a0 100644
--- a/VtNetCore/VirtualTerminal/VirtualTerminalController.cs
+++ b/VtNetCore/VirtualTerminal/VirtualTerminalController.cs
@@ -2778,8 +2778,8 @@
             SendData?.Invoke(this, new SendDataEventArgs { Data = value });
         }
 
-        private static readonly byte[] BracketedPasteModePrefix = Encoding.ASCII.GetBytes("\u001b[200~,");
-        private static readonly byte[] BracketedPasteModePostfix = Encoding.ASCII.GetBytes("\u001b[200~,");
+        private static readonly byte[] BracketedPasteModePrefix = Encoding.ASCII.GetBytes("\u001b[200~");
+        private static readonly byte[] BracketedPasteModePostfix = Encoding.ASCII.GetBytes("\u001b[201~");
 
         public void Paste(byte [] toPaste)
         {
-- 
2.17.1.windows.2

